tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -349,56 +370,42 @@ def get_version_string(self, version):
-
-        # start second with <version>
         self.prepare_for(self.processor2, version)
-        node2 = self.processor2.node
-        with node1.account.monitor_log(self.processor1.STDOUT_FILE) as first_monitor:
-            with node2.account.monitor_log(self.processor2.STDOUT_FILE) as second_monitor:
-                with node2.account.monitor_log(self.processor2.LOG_FILE) as log_monitor:
-                    self.processor2.start()
-                    log_monitor.wait_until(kafka_version_str,
-                                           timeout_sec=60,
-                                           err_msg="Could not detect Kafka Streams version " + version + " on " + str(node2.account))
-                    first_monitor.wait_until(self.processed_msg,
-                                             timeout_sec=60,
-                                             err_msg="Never saw output '%s' on " % self.processed_msg + str(node1.account))
-                    second_monitor.wait_until(self.processed_msg,
-                                              timeout_sec=60,
-                                              err_msg="Never saw output '%s' on " % self.processed_msg + str(node2.account))
-
-        # start third with <version>
         self.prepare_for(self.processor3, version)
-        node3 = self.processor3.node
-        with node1.account.monitor_log(self.processor1.STDOUT_FILE) as first_monitor:
-            with node2.account.monitor_log(self.processor2.STDOUT_FILE) as second_monitor:
-                with node3.account.monitor_log(self.processor3.STDOUT_FILE) as third_monitor:
-                    with node3.account.monitor_log(self.processor3.LOG_FILE) as log_monitor:
-                        self.processor3.start()
-                        log_monitor.wait_until(kafka_version_str,
-                                               timeout_sec=60,
-                                               err_msg="Could not detect Kafka Streams version " + version + " on " + str(node3.account))
-                        first_monitor.wait_until(self.processed_msg,
-                                                 timeout_sec=60,
-                                                 err_msg="Never saw output '%s' on " % self.processed_msg + str(node1.account))
-                        second_monitor.wait_until(self.processed_msg,
-                                                  timeout_sec=60,
-                                                  err_msg="Never saw output '%s' on " % self.processed_msg + str(node2.account))
-                        third_monitor.wait_until(self.processed_msg,
-                                                 timeout_sec=60,
-                                                 err_msg="Never saw output '%s' on " % self.processed_msg + str(node3.account))
+
+        self.processor1.start()
+        self.processor2.start()
+        self.processor3.start()
```

Comment:
I changed the "first time startup" method to just start all the instances at the same time, rather than waiting for them to start up and process one at a time.

For the upgrade part of the test, we still do rolling upgrades.